src/components/tx-flow/flows/TokenTransfer/SendToBlock.tsx

```diff
@@ -3,13 +3,13 @@ import EthHashInfo from '@/components/common/EthHashInfo'
 
 const SendToBlock = ({ address, title = 'To' }: { address: string; title?: string }) => {
   return (
-    <Grid container gap={1} alignItems="center">
+    <Grid container alignItems="center" sx={{ gap: 1 }}>
```

Comment:
Using `gap={1}` results in 0px on mobile. We might need to update this in the MUI theme.